samples/ccr/start.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
```

Comment:
Thanks for adding this! Would you mind adding a comment explaining what this file is used for and how it should be called?

Reply:
Yep. Added the description.